src/search-manager/SearchManager.ts

```diff
@@ -36,6 +36,7 @@ export interface SearchContextData {
   defaultSearchSortOrder: SearchSortOption;
   hits: ContentHit[];
   totalHits: number;
+  isLoading: boolean;
   isFetching: boolean;
```

Comment:
Do you think it is worth to remove it completely?
It is also being used in the Collection Page, which may cause the same problem there.
```suggestion
```

Reply:
Agreed.. fixed with [086f6d2](https://github.com/openedx/frontend-app-authoring/pull/1331/commits/086f6d2bd92e897446be6d31ab71a2b7bf910fe5).

>  It is also being used in the Collection Page, which may cause the same problem there.

The Collection page is now using `isLoading` from `useCollection`, so we're good there.